tests/test_loader.py

```diff
@@ -165,6 +166,24 @@ def test_yaml_load_exploit(self):
             # If the command in exploit.yaml is echoed it will return 0
             self.assertNotEqual(config, 0)
 
+    def test_env_var_regex(self):
+        test_data = [
+            {"env_var": "$OPS_DROID", "match": True},
+            {"env_var": "$OPS-DROID", "match": True},
+            {"env_var": "${OPS_DROID}", "match": True},
+            {"env_var": '"$OPSDROID_SLACK_TOKEN"', "match": True},
+            {"env_var": "$INVALID!_CHARACTERS@", "match": False},
+            {"env_var": "OPS_DROID", "match": False},
+            {"env_var": "$OPS_DROID23", "match": False},
+            {"env_var": "$UPPER-AND-lower", "match": False},
+            {"env_var": '"MISSING_PREFIX"', "match": False},
```

Comment:
Could you add a few more test cases here? Perhaps the following:

- All lowercase
- A simple sentence
- Some numbers (as a string)
- An long string with random characters including `$`s
- A `$` on its own
- An empty variable `${}`
- An empty string
- All symbols `${_}`

Reply:
Thanks for the feedback! I added more test data to cover the cases outlined above. This also led to a more exclusive regex pattern.